packages/patternfly-4/react-core/src/experimental/components/Divider/index.ts

```diff
@@ -0,0 +1 @@
+export * from '../../../components/Divider';
```

Comment:
Question.  Do we need this?  can't we export directly in packages/patternfly-4/react-core/src/experimental/components/index.ts ?

Reply:
@tlabaj technically we don't need this, as we are exporting everything from the `/experimental/components/index.ts` file you mentioned - but I've added these in for any users who are importing directly from the component file, which was part of the original feedback we got on this:

> I just got this build error...Due to specific import we have from `@patternfly/react-core/dist/js/experimental/components/Drawer;`

Reply:
Makes sense.  thanks!